config/jobs/kubernetes/sig-testing/files-remake.yaml

```diff
@@ -1,9 +1,47 @@
+periodics:
+- interval: 6h
+  kubernetes/kubernetes:
+  - cluster: k8s-infra-prow-build
```

Comment:
you'll need to set resource limits/requests to run here

Reply:
hoisted with my own petard

Reply:
do you have links to the dashboard for current resource use, and the group you need to PR into to get access?

Reply:
(need to pr this into a doc someplace)

group: https://github.com/kubernetes/k8s.io/blob/master/groups/groups.yaml#L681-L692

The only visibility we can provide right now is for the verify job as a whole (unclear which part is generated-files-remake) https://console.cloud.google.com/monitoring/metrics-explorer?project=k8s-infra-prow-build&timeDomain=6h&pageState=%7B%22xyChart%22:%7B%22dataSets%22:%5B%7B%22timeSeriesFilter%22:%7B%22filter%22:%22metric.type%3D%5C%22kubernetes.io/container/memory/used_bytes%5C%22+resource.type%3D%5C%22k8s_container%5C%22+metadata.user_labels.%5C%22prow.k8s.io/job%5C%22%3Dmonitoring.regex.full_match(%5C%22ci-kubernetes-verify.*%5C%22)%22,%22minAlignmentPeriod%22:%223600s%22,%22unitOverride%22:%22By%22,%22aggregations%22:%5B%7B%22perSeriesAligner%22:%22ALIGN_MEAN%22,%22crossSeriesReducer%22:%22REDUCE_SUM%22,%22groupByFields%22:%5B%22metadata.user_labels.%5C%22prow.k8s.io/job%5C%22%22%5D%7D,%7B%22crossSeriesReducer%22:%22REDUCE_NONE%22%7D%5D%7D,%22targetAxis%22:%22Y1%22,%22plotType%22:%22LINE%22%7D%5D,%22options%22:%7B%22mode%22:%22COLOR%22%7D,%22constantLines%22:%5B%5D,%22timeshiftDuration%22:%220s%22,%22y1Axis%22:%7B%22label%22:%22y1Axis%22,%22scale%22:%22LINEAR%22%7D%7D,%22isAutoRefresh%22:true,%22timeSelection%22:%7B%22timeRange%22:%221d%22%7D%7D

Would also probably be a good idea to exclude this out of the main verify job

Reply:
you could guess high, and then look at limit utilization metrics to see how much is actually needed